src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +83,19 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else if (rcode == SccInvalidModification)
+  {
+    OrionError oe;
+    oe.code = SccContextElementNotFound;
+    oe.reasonPhrase ="AlredyExists";
+    oe.details = "Entity alredy exists";
```

Comment:
Some PR prodecure" :) tip:

It is better to use: "Fixed in X", where X is a commit hash. That way, it is very easy for the reviewer to have a look to the commit where the fix has been addressed, without reviewing all the changes in the PR again (this is specially important in large PR).